packages/edit-post/src/components/preferences-modal/index.js

```diff
@@ -145,6 +145,13 @@ export default function EditPostPreferencesModal() {
 									label={ __( 'Display block breadcrumbs' ) }
 								/>
 							) }
+							<EnableFeature
```

Comment:
I mentioned in another issue that it'd make sense to introduce a new Accessibility panel to the settings modal instead of folding everything into appearance.

Reply:
Here: https://github.com/WordPress/gutenberg/pull/41041#issuecomment-1133632723

Reply:
Is this something that would block this PR? I believe that if at some point we introduce an a11y panel, then we can move that option there (if and when that happens)

Reply:
It shouldn't block this, just connecting the dots since it's been mentioned in a few PRs adding features but hasn't gotten much attention.